### ISSUE TITLE
🧑‍💻: Update the vanilla-js tags of the projects in `data.json` file

### ISSUE THREAD

**Issue by Avdhesh-Varshney** [Avdhesh-Varshney/WebMasterLog#1045]
:red_circle: **Title** : Update the tags of the projects in `data.json` file
:red_circle: **Enhancement Aim** : To show the tech stacks tags which are required to build that project.
:red_circle: **Changes** : 
- Update each project tags with uniform tech stacks
- Link to reach out to [`data.json`](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/src/database/data.json)

**WARNINGS:**
- Commit them in single commit.
- Follow the guidelines.
- Nothing else will be changed until permitted by admin or mentors.

> **NOTE:** Tags can be found from the each project's readme file.
> *MULTIPLE PEOPLE CAN RAISE THE PRs FOR THIS ISSUE.*
> *I WILL MENTION THEM WHICH TECH STACKS **TAGS**, THEY WILL UPDATE.*
> *THIS IS NEEDED BECAUSE DATABASE TO BE CLEAR AND PERFECT.*

### Screenshots 📷

N/A

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- What is your participant role? 

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by wildcraft958** [comment]
Hi @Avdhesh-Varshney I would like to contribute in this under KWoC 2024
Name - Animesh 
Github Link - https://github.com/wildcraft958

**Comment by Avdhesh-Varshney** [comment]
@wildcraft958 Start with the `Angular` and `Next` projects.

**Comment by Abankita** [comment]
@Avdhesh-Varshney I want to work on this. Can you assign me and specify which tags I need to work on?

**Comment by Avdhesh-Varshney** [comment]
> @Avdhesh-Varshney I want to work on this. Can you specify which tags I need to work on?

@Abankita Go with the `node` and `react` projects.

**Comment by Avdhesh-Varshney** [comment]
@wildcraft958 What's the update?

**Comment by wildcraft958** [comment]
Due to some unavoidable  circumstances  i wasn't  able to work lend me some more time i will complete it by tomorrow  EOD
Thank you.

**Comment by Sakshamjain98** [comment]
Please add the tag easy medium hard

**Comment by Avdhesh-Varshney** [comment]
> Please add the tag easy medium hard i am raising the pr for the same please add easy medium and hard tag so that i can get the points

As this issue is not closed until all the tags will not be updated. So, I'll add difficulty tag on your PR. 👍

**Comment by Sakshamjain98** [comment]
Okay 
So can I update all the tag of the data.json file
So that u can close the issue

**Comment by Sakshamjain98** [comment]
Okay 
Please merge my 404 error page pr so that can do a pr for it

**Comment by Sakshamjain98** [comment]
I am making the corrected pr for the vanila js 
please approve this under hard level and then open a new issue for the other parts and assign them to me pleasse

**Comment by Sakshamjain98** [comment]
@Avdhesh-Varshney I will update full data.json but it take my lot of time 
thats why i want u to split this into chunks and assign that issue to me

**Comment by Avdhesh-Varshney** [comment]
@Sakshamjain98 Yup, I've already warned you about the issue. But no problem, you should do with `vanilla-js` tech stack, raise another issue for another tech stacks ✅

**Comment by Sakshamjain98** [comment]
Okay then one more thing I want to confirm from u that don't commit in main 
What do u mean by this?

**Comment by Avdhesh-Varshney** [comment]
> Okay then one more thing I want to confirm from u that don't commit in main What do u mean by this?

Create your own branch, commit & raise your pr in that branch. Don't commit directly into `main` branch.

Follow point 5 [link](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md#creating-first-pull-request-)

**Comment by Sakshamjain98** [comment]
Merge the pr asap and also I have raised a new issue for another tech stack please assign that to me
